src/ray/plasma/store_exec.cc

```diff
@@ -1,3 +1,17 @@
+// Copyright 2017 The Ray Authors.
```

Comment:
Should we exclude this file? I guess this file is originated from arrow/plasma.

Reply:
I don't know. @edoakes What do you think?

Reply:
@suquark What do you think?

Reply:
Yeah let's exclude files under `plasma/`

Reply:
This file is not directly from `arrow/plasma`. But I think it might be ok to exclude all files under `plasma`